src/main/java/step2/StringAddCalculator.java

```diff
@@ -29,10 +29,10 @@ public int splitAndSum(String text) {
         }
         String delimiter = DELIMITER;
         String data = text;
-        Matcher m = CUSTOM_PATTERN.matcher(text);
-        if (m.find()) {
-            delimiter = m.group(1);
-            data = m.group(2);
+        Matcher matcher = CUSTOM_PATTERN.matcher(text);
+        if (matcher.find()) {
+            delimiter = matcher.group(1);
+            data = matcher.group(2);
```

Comment:
1,2 역시 숫자만 봐서는 뭘 의미하는지 알기 힘드죠. 매직넘버를 상수로 분리해주면 가독성을 높혀줄 수 있어요. 
```suggestion
            delimiter = matcher.group(DELIMITER_INDEX);
            data = matcher.group(DATA_INDEX);
```